core/libraries/line_item_display/EE_SPCO_Line_Item_Display_Strategy.strategy.php

```diff
@@ -401,6 +405,13 @@ private function _sub_item_row(EE_Line_Item $line_item, $options = array(), EE_L
         $html .= EEH_HTML::td();
         // end of row
         $html .= EEH_HTML::trx();
+        $html = apply_filters(
+            'FHEE__EE_SPCO_Line_Item_Display_Strategy___sub_item_row__filter_output',
```

Comment:
to more accurately follow our naming convention, this filter name should simply be:

```php
'FHEE__EE_SPCO_Line_Item_Display_Strategy___sub_item_row__html'
```